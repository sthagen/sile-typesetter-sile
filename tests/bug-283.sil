\begin[papersize=a6]{document}
\use[module=packages.retrograde,target=v0.15.0]
% I tried extensively to get this test to *fail* again prior to v0.14.15
% release but failed. Even reverting the "fixing" commit 9d6691f and a whole
% bunch of subsequent stuff I couldn't make it fail again. Apparently not only
% the initial fix but also other subsequent algorythm and processing order
% changes have added safety rails. It's possible even newer ICU libraries are
% involved in the layers of fixes. Hence I'm simplying the test on the
% assumption that the having apostrophe locations before, at, and after
% a disired break point *would* trigger this bug if we ever were to resurface
% it but that isn't confirmed. If in doubt revert this test to v0.14.14.
\neverindent
\nofolios
\font[size=32pt,language=tr]
\set[parameter=shaper.spacestretchfactor,value=2]

Yer'de İstanbuldan'sınız Bol'da.

Yer’de İstanbuldan’sınız Bol’da.

Yerde İstanbuldansınız Bolda.

\end{document}
