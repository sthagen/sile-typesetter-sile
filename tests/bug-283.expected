Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	46.5389
Set font 	Gentium Plus;32;400;;normal;;;LTR
T	60 72 85 10 71 72 w=85.8906	(Yer'de)
Mx 	118.6465
T	997 86 w=21.8906	(İs)
Mx 	140.5372
T	87 68 81 w=43.3594	(tan)
Mx 	183.8965
T	69 88 79 w=41.8750	(bul)
Mx 	225.7715
T	71 68 81 w=48.9844	(dan)
Mx 	274.7559
T	10 w=8.0000	(')
Mx 	14.8819
My 	84.9389
T	86 215 w=21.0313	(sı)
Mx 	35.9131
T	81 215 93 w=39.9063	(nız)
Mx 	84.4297
T	37 82 79 10 71 68 w=81.6875	(Bol'da)
Mx 	166.1172
T	17 w=7.3281	(.)
Mx 	14.8819
My 	123.3389
T	60 72 85 183 71 72 w=85.2188	(Yer’de)
Mx 	119.3184
T	997 86 w=21.8906	(İs)
Mx 	141.2090
T	87 68 81 w=43.3594	(tan)
Mx 	184.5684
T	69 88 79 w=41.8750	(bul)
Mx 	226.4434
T	71 68 81 w=48.9844	(dan)
Mx 	275.4278
T	183 w=7.3281	(’)
Mx 	14.8819
My 	161.7389
T	86 215 w=21.0313	(sı)
Mx 	35.9131
T	81 215 93 w=39.9063	(nız)
Mx 	84.4306
T	37 82 79 183 71 68 w=81.0156	(Bol’da)
Mx 	165.4463
T	17 w=7.3281	(.)
Mx 	14.8819
My 	200.1389
T	60 72 85 71 72 w=77.8906	(Yerde)
Mx 	115.8653
T	997 86 w=21.8906	(İs)
Mx 	137.7559
T	87 68 81 w=43.3594	(tan)
Mx 	181.1153
T	69 88 79 w=41.8750	(bul)
Mx 	222.9903
T	71 68 81 w=48.9844	(dan)
Mx 	271.9747
T	16 w=10.7813	(-)
Mx 	14.8819
My 	238.5389
T	86 215 w=21.0313	(sı)
Mx 	35.9131
T	81 215 93 w=39.9063	(nız)
Mx 	84.4410
T	37 82 79 71 68 w=73.6875	(Bolda)
Mx 	158.1285
T	17 w=7.3281	(.)
End page
Finish
